.travis.yml

```diff
@@ -26,6 +26,7 @@ matrix:
     osx_image: xcode9.2
     language: generic
 install:
+- pip install -r requirements.txt
```

Comment:
Please don't. If you want a separate target that installs dependencies via pip see: https://github.com/log2timeline/plaso/blob/master/.travis.yml

Otherwise make sure to add the dependency to GIFT PPA and l2tbinaries